### PR TITLE
Mh/create folder for engine juniors

### PR DESCRIPTION
**Pull request by mic-0-hal**
changed instructions on how to rename the file for their own upload

### CODE REVIEW

**Review by Q-Cooke — APPROVED** [review]
LGTM thanks heaps Michael

File: Tutorials/Engine team first Git Commit/.ipynb_checkpoints/introduction_to_audio_-___delete_this_and_add_your_initials_here-___-checkpoint.ipynb [hunk 1]
```diff
@@ -5,7 +5,15 @@
    "id": "125726a6",
    "metadata": {},
    "source": [
-    "# First, Rename this file by adding your initials at the end"
```
Comment (mic-0-hal) [inline Tutorials/Engine team first Git Commit/.ipynb_checkpoints/introduction_to_audio_-___delete_this_and_add_your_initials_here-___-checkpoint.ipynb:8]: not sure why this says it is a conflict

File: Tutorials/Engine team first Git Commit/.ipynb_checkpoints/introduction_to_audio_-___delete_this_and_add_your_initials_here-___-checkpoint.ipynb [hunk 2]
```diff
@@ -712,7 +725,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.8"
```
Comment (mic-0-hal) [inline Tutorials/Engine team first Git Commit/.ipynb_checkpoints/introduction_to_audio_-___delete_this_and_add_your_initials_here-___-checkpoint.ipynb:715]: how do I delete this?

**Review by mic-0-hal — COMMENTED** [review]
not sure how I make these changes

**Comment by mic-0-hal** [comment]
thanks, I created a folder that hasn't been pulled into main.
I'll make sure the students can complete this task, but can I organise some help with GitHub with you

**Pull request merged**